### PR TITLE
Add support for Uruguay

### PR DESCRIPTION
**Pull request by fabdelgado**
Added faker for Cedula de Identidad, country Uruguay

### What is the reason for this PR?

- [x] Added support for es_UY

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [x] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Review checklist

- [x] All checks have passed
- [x] Changes are approved by maintainer

### CODE REVIEW

File: test/Faker/Provider/es_UY/PersonTest.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    public function testCI()
+    {
+        //Check validation
+        self::assertTrue($this->validateCi('57351030'));
+        self::assertTrue($this->validateCi('57351046'));
+        self::assertFalse($this->validateCi('5735103-3'));
+        self::assertFalse($this->validateCi('57351033'));
+
+        //Check generation
+        self::assertTrue($this->validateCi($this->faker->ci));
```
Comment (bramceulemans) [inline test/Faker/Provider/es_UY/PersonTest.php:22]: Testing this twice does not make it more reliable. Either test it once with a seed or in a for loop.

File: src/Faker/Provider/es_UY/Person.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+     * @return int
+     */
+    public function validationDigit(string $ci): int
+    {
+        $ci = $this->cleanCi($ci);
+
+        $ci = str_pad($ci, 7, '0', STR_PAD_LEFT);
+
+        $a = 0;
+        $baseNumber = "2987634";
```
Comment (bramceulemans) [inline src/Faker/Provider/es_UY/Person.php:27]: What is this random looking number, please extract this to a constant.

**Review by bramceulemans — CHANGES_REQUESTED** [review]

**Comment by bramceulemans** [comment]
This would be great when we start extracting locales into separate extensions. Thanks for contribution though!

File: src/Faker/Provider/es_UY/Person.php (hunk 2 above)
Comment (fabdelgado) [reply src/Faker/Provider/es_UY/Person.php:27]: Ok.

File: test/Faker/Provider/es_UY/PersonTest.php (hunk 1 above)
Comment (fabdelgado) [reply test/Faker/Provider/es_UY/PersonTest.php:22]: Ok.

**Pull request closed without merging**